SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceUniqueId.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialTraceUniqueId.h"
+
+using namespace SpatialGDK;
+
+FString EventTraceUniqueId::ToString() const
+{
+	return FString::Printf(TEXT("%0X"), Hash);
+}
+
+EventTraceUniqueId EventTraceUniqueId::GenerateForRPC(Worker_EntityId Entity, uint8 Type, uint64 Id)
```

Comment:
I thought we were going to just grab an ID using an event tracing API?

Reply:
So there was some objection to this idea as it involved sending additional data which had a lot of core gdk changes - for RPCs we can actually uniquely identify the send/receive on both workers already without introducing new information!